app-config.yaml

```
@@ -67,6 +67,11 @@ proxy:
     allowedMethods: ['GET']
     auth: ${SONARQUBE_TOKEN}
 
+  '/jenkins/api':
+    target: ${JENKINS_URL}
+    headers:
+      Authorization: ${JENKINS_API_TOKEN}
```

```suggestion
      Authorization: ${JENKINS_TOKEN}
```
Rename to be consistent with other env vars